Cargo.toml

```diff
@@ -79,6 +79,12 @@ felt = { package = "cairo-felt", path = "./felt", version = "0.3.0-rc1", default
     "alloc",
 ] }
 bitvec = { version = "1", default-features = false, features = ["alloc"] }
+cairo-lang-starknet = { git = "https://github.com/starkware-libs/cairo.git", rev = "a08bedbf26b711e81cec8b7660698a3788e67d46"}
+cairo-lang-casm = { git = "https://github.com/starkware-libs/cairo.git", rev = "a08bedbf26b711e81cec8b7660698a3788e67d46" }
+
+# TODO: check these dependencies for wasm compatibility
+ark-ff = "0.4.0-alpha.7"
+ark-std = "0.3.0"
```

Comment:
We can start by using the `dep = { version = "x.y.z", default-features = false }` notation.